index.js

```diff
@@ -59,16 +59,16 @@ function mergeObject(target, source, options) {
 		})
 	}
 	getKeys(source).forEach(function(key) {
-		if (propertyIsUnsafe(target, key)) {
-			return
+			if (propertyIsUnsafe(target, key)) {
```

Comment:
Unintended indentation change?

Reply:
nice catch, should have used prettier on this project so filthy pesants like me don't make a mess <3